client/pages/item/_itemId/index.vue

```diff
@@ -41,12 +41,9 @@
             <v-row
               v-if="item && item.GenreItems && item.GenreItems.length > 0"
               align="center"
+              class="mt-sm-3"
```

Comment:
We can probably remove this, as it makes the gap between the genres and the button line really large (on all pages)